packages/salesforcedx-vscode-core/test/commands/commands.test.ts

```diff
@@ -505,4 +505,36 @@ describe('Command Utilities', () => {
       });
     });
   });
+
+  // Due to the way the prompt is phrased
+  // CONTINUE means that we will execute the forceLogoutAll command
+  // CANCEL means that we will not execute the forceLogoutAll command
+  describe('DemoModePrompGatherer', () => {
```

Comment:
Care must be taken to interpret the prompt so I've added some commands in addition to the tests.